tests/base/boring_model.py

```diff
@@ -55,8 +56,14 @@ def __len__(self):
 
 
 class BoringModel(LightningModule):
-
-    def __init__(self):
+    def __init__(
```

Comment:
let's keep the boring model simple if needed inherit and add extra argiments

Reply:
Thank you for the review! Do we keep instantiation without arguments?

Reply:
Hi @Borda, I chose to keep only in and out features as hparams to build the state of BoringModel. Other parameters are simply specified in __init__ (e.g., `optimizer_name`, ...) to avoid magic numbers and strings. Does it sound good to you?